.gitignore

```diff
@@ -10,7 +10,9 @@ umd/
 
 typedoc/
 
-*/**/yarn.lock
+examples/**/yarn.lock
```

Comment:
These are the yarn4 gitignore recommendations that I have been following in the vis.gl repos https://github.com/visgl/luma.gl/blob/master/.gitignore#L33

Reply:
I am trying different things to get netlify deployment works on this PR. So far I noticed if I don't have yarn.lock in the website and examples/demo-app folder, running yarn install in these two folders to build the website will fail with a 'issing yarn.lock' message

Reply:
Yes you should check in the yarn.lock file